_includes/documentation/documentation-version-band.html

```diff
@@ -42,6 +61,14 @@ <h3>Strimzi {{site.data.releases.operator[0].version}} documentation (latest sta
       <br/>
       <a href="{{site.baseurl}}/docs/bridge/in-development/">Strimzi Kafka Bridge documentation</a> <a href="/docs/bridge/in-development/full/bridge.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a>
       </p>
+
+      <p>
+        <strong class="text-caps">Strimzi Access Operator</strong>
+        <br/>
+        <a href="{{site.baseurl}}/docs/operators/in-development/deploying.html#assembly-access-operator-str">Managing client connections with the Access Operator</a> <a href="/docs/operators/in-development/full/deploying.html#assembly-access-operator-str" target="_blank"><i class="fas fa-external-link-alt"></i></a>
+        <a href="{{site.baseurl}}/docs/operators/in-development/deploying.html#assembly-drain-cleaner-str">Evicting pods with the Strimzi Drain Cleaner</a> <a href="/docs/operators/in-development/full/deploying.html#assembly-drain-cleaner-str" target="_blank"><i class="fas fa-external-link-alt"></i></a>
+        
+      </p>
```

Comment:
How come there isn't an in-development link for the drain cleaner?

Reply:
I didn't push up the cleaned up links. I'd removed them.